src/foam/nanos/crunch/CapabilityJunctionStatus.js

```diff
@@ -48,7 +48,7 @@ foam.ENUM({
     {
       name: 'PENDING_REVIEW',
       label: { en: 'pending review', pt: 'revisão pendente' },
-      documentation: 'The information you provided is pending signing officer review',
+      documentation: 'The information you provided is pending final review',
```

Comment:
@kennyqiyenkan why the use of 'final'?  It may suggest this is the last step, but it's granting may reveal another capability to data-fill.

Reply:
The wording came from Simon. If it is wrong, then I would suggest discussing it with them for better wording or go with what we had before.

Reply:
@RachaelDing @kennyqiyenkan Discussed with Simon and Francois and we've decided on:
"The information you provided is pending review"

Reply:
@RachaelDing Please remove the word "final". Thx

Reply:
updated. Thanks